### ISSUE TITLE
Support for smaller crosshairs?

### ISSUE THREAD

**Issue by KremlingKuthroat19** [MiSTer-devel/SNES_MiSTer#318]
Just noticed that the crosshairs for the SNES core have small and large crosshairs while many cores have small, medium and large crosshairs. What's interesting is that the same small crosshairs are labeled as medium crosshairs on other cores such as the SMS. The medium crosshairs are pretty much a + symbol.

My simple request is to add a smaller crosshair that looks like just a smaller dot * versus a + symbol for the current small crosshair.

TLDR; Can we have large, medium and small crosshair support, where large takes up the whole screen as it already does, the current small crosshair is renamed to medium and there's a new smaller crosshair dot created like it's handled in the SMS core, to take up less real estate on screen.

I know this may seem like a silly request, but it'd be nice to have crosshairs be uniform across cores if possible :)
